src/components/avatar/index.tsx

```diff
@@ -1,63 +1,37 @@
 import * as React from 'react';
 import { useTheme } from 'utils/hooks';
-import { Icon, Text } from '../../index';
+import { Text, TextSize } from '../../index';
 import { StyledAvatar, StyledAvatarImg } from './style';
 
-const getFontSize = (theme, size) => {
-  switch (true) {
-    case size < 28:
-      return theme.typography.sizes.xs;
-    case size < 34:
-      return theme.typography.sizes.s;
-    case size < 42:
-      return theme.typography.sizes.m;
-    case size < 56:
-      return theme.typography.sizes.l;
-    case size >= 56:
-      return theme.typography.sizes.xl;
-    default:
-      return theme.typography.sizes.m;
-  }
-};
-
 export interface AvatarProps {
-  alt?: string;
   children?: React.ReactNode | string;
   className?: string;
+  fontSize?: TextSize;
   imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
   size?: number;
   src?: string;
 }
 
-export const Avatar: React.FC<AvatarProps> = ({ className, src, size = 54, alt, children, imgProps }: AvatarProps) => {
+export const Avatar: React.FC<AvatarProps> = ({
+  className,
+  size = 54,
+  children,
+  imgProps,
+  src,
+  fontSize,
```

Comment:
can we still support backward compatibility for font sizes?

Reply:
We removed the `getFontSize` function, so I don't think so

Reply:
why did we remove it?

Reply:
fixed